dashboard/app/models/ability.rb

```diff
@@ -30,7 +30,14 @@ def initialize(user)
         District,
         Workshop,
         Cohort,
-        WorkshopAttendance
+        WorkshopAttendance,
+        # PLC Stuff
+        ProfessionalLearningCourse,
+        ProfessionalLearningModule,
+        ProfessionalLearningTask,
+        PlcEvaluationQuestion,
+        PlcEvaluationAnswer,
+        UserProfessionalLearningCourseEnrollment
```

Comment:
I'm kind of curious about what would happen now:
````
irb(main):001:0> module Plc
irb(main):002:1> class Module
irb(main):003:2> def a_method
irb(main):004:3>   puts "hey"
irb(main):005:3> end
irb(main):006:2> end
irb(main):007:1> end
=> :a_method
irb(main):008:0> m = Plc::Module.new
=> #<Plc::Module:0x007fda94af2f50>
irb(main):011:0> m.a_method
hey
````
hm I guess it works ok so far... still, it might be confusing should you ever want ::Module while in the Plc namespace